script.js

```diff
@@ -0,0 +1,42 @@
+const inputToDo = document.querySelector('.inputToDo');
+const buttonClick = document.querySelector('.inputButton');
+const toDoList = document.querySelector('.toDoList');
+
+const addToDo = (event) => {
+  event.preventDefault(); //submit 시 새로고침 방지
+
+  const newDiv = document.createElement('div');
```

Comment:
```suggestion
  const newTodoItemContainer = document.createElement('div');
```

변수의 이름은 이름만 보고도 그 기능을 유추할수 있도록 짓는것이 좋습니다!